package.json

```diff
@@ -56,6 +56,8 @@
     "getbase": "^2.8.2",
     "ieee754": "^1.1.8",
     "immutable": "^3.x.x",
+    "is-html": "^1.1.0",
+    "jquery": "^3.3.1",
```

Comment:
Let's avoid jQuery 😄 Consider using Cheerio instead, it will probably be a drop-in replacement